src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -290,11 +340,13 @@ public void ifPullBeforeOverridePresent(Consumer<PullBeforeOverrideOption> consu
       errors.add("Missing date range start definition");
     if (!isDateRangeValid())
       errors.add(INVALID_DATE_RANGE_MESSAGE);
+    if (formNeedsPrivateKey.orElse(false) && !isPemFilePresent())
+      errors.add(INVALID_PEM_FILE);
 
     return errors;
   }
 
-  public boolean isEmpty() {
+  public boolean isUserConfigsEmpty() {
```

Comment:
Why do we need this and the new `isEmpty()`. Is there any option to just have one definition of an empty conf? What would you need to change?